content/vsphere-cpi.md

```diff
@@ -32,6 +32,19 @@ Schema for `cloud_properties` section used by manual network subnet:
 
 * **name** [String, required]: Name of the vSphere network. Example: `VM Network`.
 
+### Managed Networks ###
+Create T1 router and attach it to T0 router. It also creates virtual switch and attaches it to T1 router.
+So every subnet in network will have its own switch. Route advertisement is enabled on T1 router.
```

Comment:
Is this paragraph intended to describe steps that the operator should take or to explain what the Managed Networks feature does? i.e. my recollection is that the T0 router is a prerequisite, but BOSH will create the T1 router.

Reply:
This is to explain what managed networks does. Should I instead update it with following description? And you are right T0 router is a prerequisite

"Automates creation of network on NSX-T. New network will have set of T1 Router and virtual switch"

Reply:
I would rephrase it to "Creates a T1 router and attaches it to T0 router. It will also create a virtual switch and attach it to the T1 router"